src/os/unix/x11.rs

```diff
@@ -90,8 +90,8 @@ struct DisplayInfo {
 }
 
 impl DisplayInfo {
-    fn new() -> Result<DisplayInfo> {
-        let mut display = Self::setup()?;
+    fn new(transparency: bool) -> Result<DisplayInfo> {
```

Comment:
I don't really like that this takes `transparency` as an argument, but I don't have any suggestions to improve this...

Reply:
The x11 one currently doesnt even work, when you try the transparency example: it produces a BadMatch on XPutImage.

Reply:
Im currently trying to find out where the problem is :/

Reply:
I actually fixed it, open to improvements now.